server.js

```diff
@@ -1,25 +1,131 @@
-import express from 'express'
-import cors from 'cors'
-import mongoose from 'mongoose'
+import express from "express"
+import cors from "cors"
+import mongoose from "mongoose"
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
+const mongoUrl =
+  process.env.MONGO_URL || "mongodb://localhost/myHappyThoughtsApi"
+mongoose.connect(mongoUrl, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+  useCreateIndex: true,
+  useFindAndModify: false,
```

Comment:
useCreateIndex and UseFindAndModify are not used in the code as far as I can see.